### PR TITLE
Pull in TLS certificates from certifi rather than relying on OS certs

### LINKED ISSUE #103: Vendor CA certificates
This way raven does not rely on system level OS certificates to be installed.

Should be easy to just vendor in https://github.com/certifi/gocertifi

### PR DESCRIPTION
**Pull request by mattrobenolt**
Fixes #103

---
This change is [Reviewable](https://reviewable.io/reviews/getsentry/raven-go/104)

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -298,9 +302,24 @@ func (c *context) interfaces() []Interface {
 // Packets will be dropped if the buffer is full. Used by NewClient.
 var MaxQueueBuffer = 100
 
+func newTransport() Transport {
+	t := &HTTPTransport{}
+	rootCAs, err := gocertifi.CACerts()
+	if err != nil {
```
Comment (philipithomas) [inline client.go]: Maybe want to check if `len(rootCAs) == 0`?

**Comment by philipithomas** [comment]
LGTM - I would just add in that you may want to add a test that the certs are valid for your domain (i.e. run a curl against https://sentry.io and make sure that it resolves)

File: client.go (hunk 1 above)
Comment (mattrobenolt) [inline client.go]: `rootCAs` isn't a slice for one. It's a `*x509.CertPool` for one. And this certificate bundle is hardcoded in https://github.com/certifi/gocertifi/blob/master/certifi.go

**Comment by mattrobenolt** [comment]
> LGTM - I would just add in that you may want to add a test that the certs are valid for your domain (i.e. run a curl against https://sentry.io and make sure that it resolves)

That would mean that our certificate became invalid for the public, which would affect every single client and every browser. We trust this certificate chain in `raven-python` already, and it's widely accepted. It's ultimately sourced from https://mkcert.org/ which is backed by Mozilla.

**Comment by mattrobenolt** [comment]
Also, it's not as simple as a curl request. It'd have to make an http request to our server from Go using this exact Transport, and I don't like automated tests that hit production services. I confirmed manually.

**Pull request merged**